core/src/main/scala/org/http4s/EntityEncoder.scala

```diff
@@ -70,7 +70,7 @@ object EntityEncoder extends EntityEncoderInstances {
     *
     * This constructor is a helper for types that can be serialized synchronously, for example a String.
     */
-  def simple[F[_], A](hs: Header*)(toChunk: A => Chunk[Byte])(
+  def simple[F[_], A](hs: Header*)(toChunk: A => Chunk[Byte])( //Fishy but I'm wondering wat to do
```

Comment:
Unsure about what to do re: A => Chunk.

Reply:
What is a use case for creating a segment here instead of a chunk?  This encoder is strict.

Reply:
hmm, the only reason I was wondering if there was anything I could do, is that toChunk currently makes me do a `segment.force` further down.